src/AppCard.test.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { render } from 'react-testing-library'
+
+import electionSample from './data/electionSample.json'
+
+import Root, { App } from './App'
+import { ActivationCardData, AdminCardData, Election } from './config/types'
+
+const election = electionSample as Election
+
+beforeEach(() => {
+  window.localStorage.clear()
+  window.location.href = '/'
+})
+
+it(`App fetches the card data every 1 second`, () => {
+  fetchMock.resetMocks()
+  jest.useFakeTimers()
+
+  fetchMock.mockResponses(
+    [JSON.stringify({}), { status: 200 }],
+    [
+      JSON.stringify({
+        present: true,
+        shortValue: JSON.stringify({
+          t: 'activation',
+          pr: election.precincts[0].id,
+          bs: election.ballotStyles[0].id,
+        }),
+      }),
+      { status: 200 },
+    ],
+    ['', { status: 500 }]
```

Comment:
Should this status be `200`? 

Where can I find more info on these status values?

Reply:
No this is meant to test that on a 500 or server error of any kind, the client bails out and thus calls `clearInterval`